### ISSUE TITLE
Create Development document - Using / adding parameter store values without using the env helm charts

### ISSUE THREAD

**Issue by patrick-black-oddball** [department-of-veterans-affairs/va.gov-team#95573]
Since we are adjusting the methodology we utilize for helm charts, the existing pattern of having different resources in different environments will no longer be supported. As a result, the VFS teams will need to know how to create and manage variables in the new system without modifying the environment helm charts. This document is needed now to ensure the development and DevOps approaches are synchronized, even though we would not release this document until the parent helm chart is moved to the master branch.

## Blocked By:

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/102127

## Tasks
- [ ] Create development document for utilizing parameter store in the new system

## Acceptance Criteria
- [ ] document created and accepted by all parties

**Comment by stevenjcumming** [comment]
Draft: https://vfs.atlassian.net/wiki/spaces/PRT/pages/edit-v2/3803513074

**Comment by stevenjcumming** [comment]
Update: Gathering feedback

**Comment by stevenjcumming** [comment]
Duplicate of https://github.com/department-of-veterans-affairs/va.gov-team/issues/102128
